login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
@alexeykazakov What should I do with the tests that used `Generate()` ? Should I remove them or replace it by another operation ?

Tests seem to work if we remove them  from their files.

```
func TestValidOAuthAccessToken(t *testing.T) {...
```

```
func TestGenerateToken(t *testing.T) {..
```